resources/user-agents/browsers/yabrowser/yabrowser-1-20-14-12.json

```diff
@@ -269,6 +269,68 @@
           ]
         }
       ]
+    },
+    {
+      "userAgent": "Yandex Browser #MAJORVER#.#MINORVER# for iOS",
```

Comment:
The oldest version I can find in our logs is 13.9, which is the oldest version in this file (if we exclude that odd `1.20` version) so I think it should be fine to keep the iOS versions in this file.

Here's all of the different versions for YaBrowser iOS that we've seen:

```
13.9
13.11
13.12
14.1
14.2
14.4
14.5
14.7
14.8
14.12
15.1
15.2
15.4
15.6
15.7
15.9
15.12
16.1
16.2
16.4
16.6
16.7
16.9
16.11
17.1
17.3
17.4
```